ansible/group_vars/server

```diff
@@ -0,0 +1,3 @@
+nfs_fstype: "xfs"
```

Comment:
Shouldn't this file be named `nfs_server` rather than `server`?

Reply:
Groups `server` and `nfs_server` should both be created.

From jasmin-cluster-infra:
```
group_inventory_groups: 
      - cluster
      - "{{ [cluster_name, group.name] | join('_')}}"
      - "{{ group.name }}"
```

Is one preferable over the other?

Reply:
My thought was that, since this group_vars file potentially applies to all cluster deployed, naming it more specifically avoids confusion.

Reply:
Sounds reasonable to me. Will change.